examples/linear_model/plot_lasso_model_selection.py

```diff
@@ -54,6 +54,8 @@
 from sklearn.linear_model import LassoCV, LassoLarsCV, LassoLarsIC
 from sklearn import datasets
 
+EPSILONE = 1e-4
```

Comment:
to fix division by zero in `log10`

Reply:
Usually this would be `EPSILON` or `EPS`